packages/core/src/types/type-info.ts

```diff
@@ -14,7 +14,7 @@ export type BaseListTypeInfo<Session = any> = {
     create: GraphQLInput
     update: GraphQLInput
     where: GraphQLInput
-    uniqueWhere: { readonly id?: string | number | null } & GraphQLInput
+    uniqueWhere: { readonly id?: unknown } & GraphQLInput
```

Comment:
`string | number | null` covers all that the `ID` GraphQL type could be, what does widening this help?